packages/web/src/scss/components/TextArea/_theme.scss

```diff
@@ -1 +1,2 @@
 $input-min-height: 3.375rem;
+$input-max-rows: 15;
```

Comment:
Could you please mention this behaviour in the [README](https://github.com/lmc-eu/spirit-design-system/pull/823/files#diff-66e47c4590f09b83d52f3578da3c76532046838e54b481c4f4d1babe9ad47321R64)?
During implementation of the `TextArea` there was UX requirement not to set maximal height of the area, so this should be BC?